docs/_documentations/eclipse-firstproject.md

```diff
@@ -43,14 +43,13 @@ This icon launches your web browser and displays your application.
 
 Congratulations, you have now created, built and run your first Codewind project using Codewind on your local machine.
 
+## Next Steps
 
-# Next Steps
+Choose from the following:
 
-For next steps, choose from the following:
+**Getting started with Codewind - next steps** Continue to instructions for [Making a Code change with Eclipse](./eclipse-codechange.html) to see the auto-build-and-run feature in action.
 
-* If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
+**Installing Codewind for remote use** If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
```

Comment:
In an earlier version of this file, the sentences with the links appeared as a list. However, without `-`, they'll just appear as normal lines of text. If we want that, then that's OK, just wanted to make sure since it changed.

Reply:
"Prerequisite" doesn't need a hyphen.

Reply:
Could we make the sentence a complete sentence? (Style guide specifications here: https://github.com/eclipse/codewind-docs/wiki/Writing-style-guidelines#colons.) Maybe something as simple as removing the "to" and saying, "... and can proceed:"?